examples/mobile/UIComponents/components/units/fonts.html

```diff
@@ -0,0 +1,133 @@
+				<div style="font-size: 24px;">Roboto-Regular 24px</div>
+				<div style="font-size: 25px;">Roboto-Regular 25px</div>
+				<div style="font-size: 26px;">Roboto-Regular 26px</div>
+				<div style="font-size: 27px;">Roboto-Regular 27px</div>
+				<div style="font-size: 28px;">Roboto-Regular 28px</div>
+				<div style="font-size: 29px;">Roboto-Regular 29px</div>
+				<div style="font-size: 30px;">Roboto-Regular 30px</div>
+				<div style="font-size: 31px;">Roboto-Regular 31px</div>
+				<div style="font-size: 32px;">Roboto-Regular 32px</div>
+				<div style="font-size: 33px;">Roboto-Regular 33px</div>
+				<div style="font-size: 34px;">Roboto-Regular 34px</div>
+				<div style="font-size: 35px;">Roboto-Regular 35px</div>
+				<div style="font-size: 36px;">Roboto-Regular 36px</div>
+				<div style="font-size: 37px;">Roboto-Regular 37px</div>
+				<div style="font-size: 38px;">Roboto-Regular 38px</div>
+				<div style="font-size: 39px;">Roboto-Regula 39px</div>
+				<div style="font-size: 40px;">Roboto-Regula 40px</div>
+			</div>
+			<br>
+
+            Font Name : Roboto - Medium
+			<div style="font-family: Roboto-Medium">
+				<div style="font-size: 10px;">Roboto-Medium 10px</div>
+				<div style="font-size: 11px;">Roboto-Medium 11px</div>
+				<div style="font-size: 12px;">Roboto-Medium 12px</div>
+				<div style="font-size: 13px;">Roboto-Medium 13px</div>
+				<div style="font-size: 14px;">Roboto-Medium 14px</div>
+				<div style="font-size: 15px;">Roboto-Medium 15px</div>
+				<div style="font-size: 16px;">Roboto-Medium 16px</div>
+				<div style="font-size: 17px;">Roboto-Medium 17px</div>
+				<div style="font-size: 18px;">Roboto-Medium 18px</div>
+				<div style="font-size: 19px;">Roboto-Medium 19px</div>
+				<div style="font-size: 20px;">Roboto-Medium 20px</div>
+				<div style="font-size: 21px;">Roboto-Medium 21px</div>
+				<div style="font-size: 22px;">Roboto-Medium 22px</div>
+				<div style="font-size: 23px;">Roboto-Medium 23px</div>
+				<div style="font-size: 24px;">Roboto-Medium 24px</div>
+				<div style="font-size: 25px;">Roboto-Medium 25px</div>
+				<div style="font-size: 26px;">Roboto-Medium 26px</div>
+				<div style="font-size: 27px;">Roboto-Medium 27px</div>
+				<div style="font-size: 28px;">Roboto-Medium 28px</div>
+				<div style="font-size: 29px;">Roboto-Medium 29px</div>
+				<div style="font-size: 30px;">Roboto-Medium 30px</div>
+				<div style="font-size: 31px;">Roboto-Medium 31px</div>
+				<div style="font-size: 32px;">Roboto-Medium 32px</div>
+				<div style="font-size: 33px;">Roboto-Medium 33px</div>
+				<div style="font-size: 34px;">Roboto-Medium 34px</div>
+				<div style="font-size: 35px;">Roboto-Medium 35px</div>
+				<div style="font-size: 36px;">Roboto-Medium 36px</div>
+				<div style="font-size: 37px;">Roboto-Medium 37px</div>
+				<div style="font-size: 38px;">Roboto-Mediu 38px</div>
+				<div style="font-size: 39px;">Roboto-Mediu 39px</div>
+				<div style="font-size: 40px;">Roboto-Mediu 40px</div>
```

Comment:
m missing

Reply:
If I fix it then line will be wrapped.
I can add overflow:hidden style but it will be cut.

Reply:
how about just using abbreviation of Medium in this case i.e. "Med."